app/helpers/flights_helper.rb

```diff
@@ -5,7 +5,7 @@ def get_user_image(user_image)
   end
 
   def get_all_airports(field_name, msg, class_desc)
-    collection_select :flight, field_name, Airport.all, :id, :name, {include_blank: msg}, {class: class_desc}
+    collection_select :flight, field_name, Airport.order("name ASC"), :id, :name, {include_blank: msg}, {class: class_desc}
```

Comment:
Line is too long. [123/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.